File: Step1/Logical-Thinking/Pattern13/Solution.cpp
#include <iostream>
using namespace std;

void pattern(int n){
    int num = 1;
    for(int i = 1; i<=n; ++i){
        for(int j = 1; j<=i; ++j){
            cout<<num<<" ";
            ++num;
        }
        cout<<endl;
    }
}
int main()
{
    pattern(6);

    return 0;
}

File: Step1/Logical-Thinking/Pattern13/Solution.java
// SAMPLE PATTERN
// 1 
// 2 3
// 4 5 6
// 7 8 9 10
// 11 12 13 14 15
public class Main
{
    static void pattern(int n){
    int num = 1;
    for(int i = 1; i<=n; ++i){
        for(int j = 1; j<=i; ++j){
            System.out.print(num+" ");
            ++num;
        }
        System.out.println();
    }
}
	public static void main(String[] args) {
		pattern(5);
	}
}
